database/generateData.js

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable func-names */
+const faker = require('faker');
+const db = require('./index');
+
+// generate fake data for 100 unique businesses
+const generateBusiness = function () {
+  const businesses = [];
+  for (let i = 0; i < 100; i += 1) {
+    const entry = {
+      name: faker.company.companyName(),
+      claimed: Math.floor(Math.random() * Math.floor(2)),
+      // overallRating needs future refactor to generate dynamic rating
+      overallRating: Math.floor(Math.random() * Math.floor(5)) + 1,
+      // totalReviews needs future refactor to generate dynamic count
+      totalReviews: Math.floor(Math.random() * Math.floor(60)) + 10,
+      averageCost: Math.floor(Math.random() * Math.floor(4)) + 1,
+      businessType: `${faker.company.catchPhraseAdjective()}, ${faker.name.jobArea()}`,
+      address1: `${faker.address.streetAddress()}`,
+      address2: `${faker.address.city()}, ${faker.address.stateAbbr()} ${faker.address.zipCode()}`,
+      address3: `b/t ${faker.address.streetName()} ${faker.address.streetSuffix()} & ${faker.address.streetName()} ${faker.address.streetSuffix()}`,
+      address4: `${faker.address.city()}`,
+      phoneNumber: faker.phone.phoneNumber(),
+      url: `${faker.lorem.word()}${faker.internet.domainWord()}.${faker.internet.domainSuffix()}`,
```

Comment:
love the set up the url, great work Daniel

Reply:
Thanks!